src/main/java/com/carmen/app/boundary/ICarResource.java

```diff
@@ -0,0 +1,80 @@
+package com.carmen.app.boundary;
+
+import java.util.List;
+import java.util.UUID;
+
+import javax.ws.rs.core.Response;
+
+import com.carmen.app.entities.Car;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.parameters.RequestBody;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+
+public interface ICarResource {
+
+	@Operation(summary = "Get all cars",
+			responses = {
+					@ApiResponse(responseCode = "200",
+							description = "Get all cars",
+							content =  @Content(mediaType = "application/json",
+							schema = @Schema(implementation = Car.class)))
+			})
+public List<Car> getCars() ;
+	
+	@Operation(summary = "Get car by id",
+			responses = {
+					@ApiResponse(responseCode = "200",
+							description = "Get car by id",
+							content = @Content(mediaType = "application/json",
+							schema = @Schema(implementation = Car.class))),
+					@ApiResponse(responseCode = "404",
+								description = "There is no car with this id")
+			})
+public Response getCar(@Parameter(description = "Car id", required = true) UUID id);
```

Comment:
En general is important to maintain a good code styling. Could you please check the code indentation.
In eclipse to maintain it uniform you can use the shortcut CTRL + Shift + F .

Looks like there is a way to make it automatic when saving the files: https://stackoverflow.com/a/21485725/9539862